edgelet/iotedged/src/lib.rs

```diff
@@ -515,14 +518,25 @@ where
     let (mgmt_tx, mgmt_rx) = oneshot::channel();
     let (work_tx, work_rx) = oneshot::channel();
 
-    let mgmt = start_management(&settings, &runtime, &id_man, mgmt_rx);
+    let edgelet_cert_props = CertificateProperties::new(
+        IOTEDGED_VALIDITY,
+        IOTEDGED_TLS_COMMONNAME.to_string(),
+        CertificateType::Server,
+        "iotedge-tls".to_string(),
+    )
+    .with_issuer(CertificateIssuer::DeviceCa);
```

Comment:
It'll be nice to have these parameters, i.e., validity, TLS name and alias come from configuration or an env variable. That way customers can override it. But we can do that via a different PR and just file a task for this.

Reply:
@avranju created task # 4318495